queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py

```diff
@@ -109,6 +126,20 @@ def _get_pdfs(
             attach_order += 1
 
     elif status == Filing.Status.COMPLETED.value:
+        # add filing pdf
+        filing_pdf_type = 'continuationIn'
+        filing_pdf_encoded = get_filing_document(business['identifier'], filing.id, filing_pdf_type, token)
+        if filing_pdf_encoded:
+            pdfs.append(
+                {
+                    'fileName': 'Continuation Application.pdf',
+                    'fileBytes': filing_pdf_encoded.decode('utf-8'),
+                    'fileUrl': '',
+                    'attachOrder': str(attach_order)
+                }
+            )
+            attach_order += 1
+
```

Comment:
From my understanding continuation application document is not required in COMPLETED status

Reply:
It's currently showing in the email's attachment list:
![image](https://github.com/user-attachments/assets/d8cb3ede-8c9d-480f-b229-4db55d92080a)

If you can confirm this, I can update the email template, so people won't get confused about the listed items and the attachments they actually get.

Reply:
I just looked into design. This code is based on design (https://www.figma.com/design/rNe2jn8wyFykKRzwiBo6ar/Continuation-In-V6?node-id=905-121&node-type=frame&t=efGzao6GeDDTAWup-0) but its not aligned with any other filings. So I assume this was a misunderstanding

Reply:
This contains all the email templates https://docs.google.com/spreadsheets/d/17wMY9znmWMr2RoXH9S2elffC8uE7iifUGB5Hu3VVFpE/edit?gid=0#gid=0

Reply:
Can you discuss this with Mihai/Jacqueline

Reply:
yes, was about to talk to Jacqueline. Thank you for pointing it out.

Reply:
Then this question should go back to @jacqueline-williams-549 .

Reply:
The continuation application output should show in the email once the user submits a future effective application or for an application that is registered immediately. Is there an instance where the continuation is registered immediately by the user but not 'recognized' yet by registrar? If that is the case there could be a design missing from the figma link that you have. @eason-pan-bc

Reply:
So, there are 2 emails sent out. 
1. Before the filing has been processed:
![image](https://github.com/user-attachments/assets/23db5506-06c6-4c37-bbb4-00b2795033e5)
2. After the application is completed 
![image](https://github.com/user-attachments/assets/3997cae7-a94e-43ba-8f35-15894ddfb2ed)

The receipt and continuation application pdf question is regarding the second one.

Reply:
Okay thank you, we can keep it that way 

1. User completed payment (PAID) - We attach receipt and the continuation application in this email
2. System processed the filing (COMPLETED) - NOA and Certificate attached in this email


I've updated the designs here: https://www.figma.com/design/rNe2jn8wyFykKRzwiBo6ar/Continuation-In-V6?node-id=201-2&m=dev&focus-id=1978-13162

Reply:
template updated and not attaching the 2 documents for the COMPLETED status email.